keras/layers/layer.py

```diff
@@ -646,6 +648,22 @@ def metrics_variables(self):
                 vars.extend(metric.variables)
         return vars
 
+    @property
+    def seed_generators(self):
```

Comment:
It isn't intended to be accessed by users, let's not make it public. If a user needs to access the RNG state variable for a specific layer they could it via public APIs, like `layer.seed_generator.state`.

Reply:
Got it. I have removed it and used an internal method `get_seed_generators`  in `testing.test_case.py` to get the correct number of seed generators.